libs/ob-sql-parser/src/main/resources/oboracle/sql/OBParser.g4

```diff
@@ -826,7 +848,7 @@ func_param
     ;
 
 func_param_with_assign
-    : var_name PARAM_ASSIGN_OPERATOR (bit_expr|bool_pri_in_pl_func)
+    : pl_var_name PARAM_ASSIGN_OPERATOR (bit_expr|bool_pri_in_pl_func)
```

Comment:
further more, expression 'updating_params' should be updated

Reply:
The previous `updating_params` is `STRING_VALUE | pl_var_name` which is more lenient than the current one(`bit_expr`). I didn't update the `updating_params` for compatibility with older OB versions.

Reply:
actually, bit_expr is the most lenient expression
well, It is okay this way~